### DISCUSSION TITLE
Smoothen the scrollintoview?

### DISCUSSION THREAD

**Discussion by skimaharvey** [idiotWu/smooth-scrollbar#334] [Q&A] [answered]
Hi,

Is there anyway to smoothen the scrollintoview without changing the damping?

Thank you!

**Comment by dolphin-wood** [comment, marked as answer]
You can uae the [`scrollTo`](https://github.com/idiotWu/smooth-scrollbar/blob/develop/docs/api.md#scrollbarscrollto) method with a custom easing function.
